controllers/v1/project/templateTasks.js

```diff
@@ -139,11 +138,11 @@ module.exports = class ProjectTemplateTasks extends Abstract {
     }
 
     /**
-    * @api {post} /improvement-project/api/v1/project/templateTasks/update/:taskId 
+    * @api {post} /project/v1/project/templateTasks/update/:taskId 
     * Update projects template.
     * @apiVersion 1.0.0
     * @apiGroup Project Template Tasks
-    * @apiSampleRequest /improvement-project/api/v1/project/templateTasks/update/6006b5cca1a95727dbcdf648
+    * @apiSampleRequest /project/v1/project/templateTasks/update/6006b5cca1a95727dbcdf648
```

Comment:
@Prajwal17Tunerlabs add req body for this api

Reply:
we are not using this API, and also this API is not mentioned in one big thing, so i dont know the api doc for this